src/rubocop/OffenseFormatter.js

```diff
@@ -0,0 +1,57 @@
+'use babel'
+
+import { satisfies } from 'semver'
+import getRuleDocumentation from './helpers/documentation-cache'
+
+import ErrorFormatter from '../ErrorFormatter'
+
+const SEVERITY_MAPPING = {
+  refactor: 'info',
+  convention: 'info',
+  warning: 'warning',
+  error: 'error',
+  fatal: 'error',
+}
+
+const HASCOPNAME_VERSION_RANGE = '>=0.52.0 <0.68.0'
+const RULE_MATCH_REGEX = /https:\/\/.*#(.*)/g
+
+export default class OffenseFormatter extends ErrorFormatter {
+  format(version, {
+    message: rawMessage, location, severity, cop_name: copName,
+  }, filePath) {
+    const hasCopName = satisfies(version, HASCOPNAME_VERSION_RANGE)
+    const [excerpt, url] = rawMessage.split(/ \((.*)\)/, 2)
+    let position
+    if (location) {
+      const { line, column, length } = location
+      position = [[line - 1, column - 1], [line - 1, (length + column) - 1]]
+    } else {
+      position = this.topFileRange
+    }
+
+    const linterMessage = {
+      url,
```

Comment:
🔥 Don't specify the `url` property unless you actually have a value to stick in there.